src/screens/Library/AllGroups/AllGroups.jsx

```diff
@@ -126,6 +126,11 @@ const AllGroups = () => {
           </Link>
         ),
       },
+      {
+        Header: t("screens.allGroups.groupDescription"),
+        accessor: "description",
+        Cell: ({ row }) => <>{row.original.description}</>,
+      },
```

Comment:
maybe we can cut the text with `...` at the end because this doesnt looks good.
![image](https://user-images.githubusercontent.com/60048565/117478855-f8736c80-af5f-11eb-9d8c-2229c47f7636.png)

Reply:
I would say we limit the description length, and add `...` with hover